apps/main/src/loan/components/PageLlamaMarkets/columns.tsx

```diff
@@ -0,0 +1,107 @@
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { ColumnDef, ColumnFiltersState, createColumnHelper, FilterFnOption } from '@tanstack/react-table'
+import { LlamaMarket, LlamaMarketType } from '@/loan/entities/llama-markets'
```

Comment:
`LlamaMarketType` unused import as well

Reply:
I'm not sure I agree, because unused imports won't break anything. Eventually we see the warnings and run `--fix`